walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.json

```diff
@@ -12,71 +12,134 @@
-            "description": "Basic CRUD application for fruit"
+            "openshift.io/display-name": "RHMI Lab Node.js Frontend & API",
+            "description": "The external order UI and API that's used in RHMI Solution Patterns",
+            "iconClass": "icon-nodejs",
+            "tags": "rhmi,nodejs"
           }
         },
         "objects": [
           {
+            "apiVersion": "v1",
             "kind": "DeploymentConfig",
-            "apiVersion": "apps.openshift.io/v1",
             "metadata": {
-              "name": "crud",
               "labels": {
-                "app": "crud"
-              }
+                "app": "rhmi-lab-nodejs-order-frontend"
+              },
+              "name": "rhmi-lab-nodejs-order-frontend"
             },
             "spec": {
               "replicas": 1,
-              "revisionHistoryLimit": 10,
-              "test": false,
               "selector": {
-                "app": "crud"
+                "app": "rhmi-lab-nodejs-order-frontend",
+                "deploymentconfig": "rhmi-lab-nodejs-order-frontend"
+              },
+              "strategy": {
+                "activeDeadlineSeconds": 21600,
+                "resources": {},
+                "rollingParams": {
+                  "intervalSeconds": 1,
+                  "maxSurge": "25%",
+                  "maxUnavailable": "25%",
+                  "timeoutSeconds": 600,
+                  "updatePeriodSeconds": 1
+                },
+                "type": "Rolling"
               },
               "template": {
                 "metadata": {
                   "labels": {
-                    "app": "crud"
+                    "app": "rhmi-lab-nodejs-order-frontend",
+                    "deploymentconfig": "rhmi-lab-nodejs-order-frontend"
                   }
                 },
                 "spec": {
                   "containers": [
                     {
-                      "name": "crud",
-                      "image": "quay.io/integreatly/fruit-crud-app:1.0.2",
+                      "env": [
+                        {
+                          "name": "MESSAGING_SERVICE_HOST",
+                          "value": "${MESSAGING_SERVICE_HOST}"
+                        },
+                        {
+                          "name": "MESSAGING_SERVICE_USER",
+                          "value": "${MESSAGING_SERVICE_USER}"
+                        },
+                        {
+                          "name": "MESSAGING_SERVICE_PASSWORD",
+                          "value": "${MESSAGING_SERVICE_PASSWORD}"
+                        },
+                        {
+                          "name": "MESSAGING_SERVICE_PORT",
+                          "value": "${MESSAGING_SERVICE_PORT}"
+                        }
+                      ],
+                      "image": "quay.io/evanshortiss/rhmi-lab-nodejs-order-frontend",
```

Comment:
There are three instances where we reference images in `quay.io/evanshortiss/` in this PR. These need to be updated to fall under the integr8ly org in quay.io